### ISSUE TITLE
sso_session config doesn't work with the plugin

### ISSUE THREAD

**Issue by jesse-r2c** [mhassan1/yarn-plugin-aws-codeartifact#38]
Hey all, thanks for creating this plugin! 

## Issue
We're using `v0.16.1` of the plugin in our app and I ran into an issue with the sso authentication using the [sso_session section ](https://docs.aws.amazon.com/cli/latest/userguide/sso-configure-profile-token.html#sso-configure-profile-token-manual)of the aws config. 

Most of the devs working on the app have an aws config that looks something like this:
```
[profile dev]
sso_session = my-sso
sso_account_id = 111122223333
sso_role_name = SampleRole

[sso-session my-sso]
sso_region = us-east-1
sso_start_url = https://my-sso-portal.awsapps.com/start
```

This doesn't seem to work with the plugin. Changing the config to this fixes the issue:
```
[profile dev]
sso_region = us-east-1
sso_start_url = https://my-sso-portal.awsapps.com/start
sso_account_id = 111122223333
sso_role_name = SampleRole
```

`v0.16.1` and the current version are both using `aws-sdk/*@3.363.0` so I'm assuming that the issue also exists in the current version although I can't confirm that at the moment. 

## Workaround

We have a workaround in place, but ideally we wouldn't have to have 80+ engineers updating their aws configs to get this to work:

```
# ~/.aws/config
[profile dev-legacy]
sso_region = us-east-1
sso_start_url = https://my-sso-portal.awsapps.com/start
sso_account_id = 111122223333
sso_role_name = SampleRole

[profile dev]
sso_session = my-sso
sso_account_id = 111122223333
sso_role_name = SampleRole

[sso-session my-sso]
sso_region = us-east-1
sso_start_url = https://my-sso-portal.awsapps.com/start

# .yarn-plugin-codeartifact.yml
...
  awsProfile: dev-legacy
  ```

Hopefully the resolution is as easy as bumping the aws-sdk versions and calling it a day 🤞 

Thanks again for providing this plugin, it's significantly streamlined our auth process!

**Comment by mhassan1** [comment]
I have upgraded `@aws-sdk/*` on a branch; please try `yarn plugin import https://raw.githubusercontent.com/mhassan1/yarn-plugin-aws-codeartifact/bump-aws-deps/bundles/@yarnpkg/plugin-aws-codeartifact.js` and let me know if it resolves the issue. If it does, I will merge that PR.

**Comment by jesse-r2c** [comment]
@mhassan1 Sorry for the late response, this worked for me! Thank you for taking a look at this! 🥳

**Comment by mhassan1** [comment]
This should be resolved on `v0.18.0` of the plugin.
